ivle-sync.py

```diff
@@ -229,7 +288,7 @@ def main():
         print("Aborting...")
         exit(-1)
 
-    print("Usage: " + argv[0] + " [files|announcements]")
+    print("Usage: " + argv[0] + " [files|announcements|clear-token]")
```

Comment:
`clear-token` sounds a little ambiguous to me. Would `logout` or `reset` work better?